test/nodejs/route.js

```diff
@@ -5,6 +5,43 @@ var monaco_mld_path = require('./constants').mld_data_path;
 var monaco_corech_path = require('./constants').corech_data_path;
 var three_test_coordinates = require('./constants').three_test_coordinates;
 var two_test_coordinates = require('./constants').two_test_coordinates;
+const flatbuffers = require('../../features/support/flatbuffers').flatbuffers;
```

Comment:
Here I had to import flatbuffers related things from `features` folder, IMO it is a bit ugly, but not a big problem - as an alternative we probably can re-arrange folders somehow or add symlink to that folder in `test/nodejs`.